ccd-definition/AuthorisationCaseField/la-barrister.json

```diff
@@ -0,0 +1,429 @@
+[
```

Comment:
have some qs about all the fields the user has access to... I can see they only have access to upload documents event, so do I need all these C U permissions?

Reply:
will change this to R and test it again, good point :-)